### PR TITLE
BZ #860428: remove realm selection from deployment overview page

### PR DESCRIPTION
**Pull request by ifarkas**
https://bugzilla.redhat.com/show_bug.cgi?id=860428

### CODE REVIEW

**Comment by jcoufal** [comment]
Partial ack. Process is OK now, just two small notes:
- Don't allow to change deployment name in Overview - display it.
- I named deployment "mock_deployment_2" and in Overview I got "mock-deployment-2" (with dashes), so we modified the name user chose. We should't do that.

**Comment by ifarkas** [comment]
> - Don't allow to change deployment name in Overview - display it.
> - I named deployment "mock_deployment_2" and in Overview I got "mock-deployment-2" (with dashes), so we modified the name user chose. We should't do that.

@jcoufal Have you checked the linked BZ ticket? They specifically asked to remove the realm selection. While I agree that there's a lot needs to be changed regarding deployment launching but since this should also go to 1.1, I would keep this patch as compact and simple as it can be. 
I would prefer to have the deployment launch refactoring during some feature implementation and not with some bugfixing patch.

File: src/app/views/deployments/_overview.html.haml [hunk 1]
```diff
@@ -24,7 +24,7 @@
         .field
           = label_tag :name, t('.name'), :class => 'em'
```
Comment (jcoufal) [inline src/app/views/deployments/_overview.html.haml:27]: With change for .text_instead_input, the class for label should change to .field according to similar rows. (it is affecting padding, which is anyway wrong, but it should be at least same at all fields)

**Comment by jcoufal** [comment]
@ifarkas Yeah, I did look and your previous patch fixed the BZ ticket. I just saw related small issue - that you are able to change name in overview as well, so I think that it should be corrected too. Anyway thank you for fixing that, even though it was not your task.

Just one little correction in above comment.

2 cucumber tests failed :(
Failing Scenarios:
cucumber features/deployment.feature:29 # Scenario: Launch new deployment
cucumber features/deployment.feature:49 # Scenario: Launch new deployment over X

**Comment by ifarkas** [comment]
@jcoufal Ahh, I was mainly talking about the name conversion (underscore vs dashes) which I am not really aware if it is a feature or bug and what would be the effect of changing it, I would rather not risk to change it as it is also going to 1.1 branch.
Sorry for the confusion!

jtomasek suggested that instead of removing the realm selection, we could display it as a text, so I changed that. The same for the deployment name as you suggested. 
The failing tests are also fixed.

> With change for .text_instead_input, the class for label should change to .field according to similar rows. (it is affecting padding, which is anyway wrong, but it should be at least same at all fields)

Could you explain this a bit more? As I can see, in the other rows the labels don't have .field class but the label and the input field are wrapped in a div with a class .field, just as mine. What am I missing here?

**Comment by sseago** [comment]
ACK this seems to do what the bug report calls for. It's a bit awkward if there's a matching error -- you get a "confirmation" page with a disabled 'launch' and nothing to do but go back or cancel, but it was this way before, too. Before it was worse, since you had a form with form elements and no way to submit them.

**Comment by ifarkas** [comment]
@jcoufal, to address your concerns about the styling I created a new pull request: https://github.com/aeolusproject/conductor/pull/137

**Pull request merged**